packages/terra-core-docs/src/terra-dev-site/doc/progress-bar/example/ProgressBarDefault.jsx

```diff
@@ -25,8 +25,8 @@ const ProgressBarDefault = () => {
 
   return (
     <div>
-      <label aria-live="polite">{`Progress bar: ${val}%`}</label>
```

Comment:
We need to keep this example as a reference for using progressbar with Label (as this example was created as per the A11y team input) and update the other examples to use ValueText prop.

Reply:
yes. have updated all other examples. 
Thanks

Reply:
ValueText Prop is still being used in this example

Reply:
Yes @supreethmr .
![Screenshot 2024-02-06 at 12 29 12 PM](https://github.com/cerner/terra-core/assets/132886427/e85ca8fc-be7e-4766-af1c-2a5c627efa4e)